js/navigation/NavigationHelper.js

```diff
@@ -2,7 +2,6 @@ import { NavigationActions } from "@expo/ex-navigation";
 import Store from "../redux/store";
```

Comment:
There's a minor merge conflict in this file (against develop). Please resolve and I will approve!